### PR TITLE
Feat/customer detail service

### PR DESCRIPTION
**Pull request by yjwrjh**
please review this code that was made for the input of customer details when they book a seat

### CODE REVIEW

File: src/main/java/com/distinction/barbenheimer/controller/CustomerDetailController.java
Comment (EddyEhhh) [inline src/main/java/com/distinction/barbenheimer/controller/CustomerDetailController.java:1]: Customer details will not require a controller layer. The service will be utilised by another component. 😄

**Review by EddyEhhh — CHANGES_REQUESTED** [review]
1 issue to review!

File: src/main/java/com/distinction/barbenheimer/controller/CustomerDetailController.java
Comment (yjwrjh) [reply src/main/java/com/distinction/barbenheimer/controller/CustomerDetailController.java:1]: issue resolved, file deleted.

Comment (EddyEhhh) [reply src/main/java/com/distinction/barbenheimer/controller/CustomerDetailController.java:1]: Awesome thank you! 😄

**Pull request merged**